test/FadeMixinSpec.js

```diff
@@ -54,4 +55,23 @@ describe('FadeMixin', function () {
       }, 25);
     }, 25);
   });
+
+  it('Should be b of the textarea value', function (done) {
```

Comment:
Something to the effect of `should retain the same input value throughout transition` would be more clear. The test name should describe your intent, not the implementation details.